Dockerfiles/midstream/unit_tests.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import unittest
+import os
+import subprocess
+
+
+class Testing(unittest.TestCase):
+
+    def setUp(self):
+        exec_cmd = "podman build -t midstream:latest -f Dockerfile"
+        subprocess.run(exec_cmd, shell=True)
+
+    def test_postive(self):
+        exec_cmd = """
+        podman run -it -v ../../:/project/operator-test-playbooks -v ../.././output_logs:/project/output midstream:latest
+        """
+        result = subprocess.run(exec_cmd, shell=True)
+        self.assertEqual(0,result.returncode)
+
+    def test_negative_55(self):
```

Comment:
Do not use numbers in the name, but rather use name of test-case which you want to cover.
e.g. extract operator bundle failed, operator sdk failed ...

Reply:
Tests renamed.